packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1413,9 +1413,7 @@ class ResizeImage extends ImageProvider<ResizeImageKey> {
 ///
 /// When a network image is used on the Web platform, the `getTargetSize`
 /// parameter of the [ImageDecoderCallback] is only supported when the
-/// application is running with the CanvasKit renderer. When the application is
-/// using the HTML renderer, the web engine delegates image decoding of network
-/// images to the Web, which does not support custom decode sizes.
+/// application is running with the CanvasKit renderer.
```

Comment:
Which I think now simply means it's always supported, and we don't need this paragraph at all.